File: GTTS/INL.TrainingService.Models/TrainingEventStudent_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.TrainingService.Models
{
    public class TrainingEventStudent_Item : ITrainingEventStudent_Item
    {
        public long PersonID { get; set; }
        public long TrainingEventID { get; set; }
        public bool? IsVIP { get; set; }
        public bool? IsParticipant { get; set; }
        public bool? IsTraveling { get; set; }
        public int? DepartureCityID { get; set; }
        public DateTime? DepartureDate { get; set; }
        public DateTime? ReturnDate { get; set; }
        public int? VisaStatusID { get; set; }
        public bool? HasLocalGovTrust { get; set; }
        public DateTime? LocalGovTrustCertDate { get; set; }
		public bool? OtherVetting { get; set; }
        public bool? PassedOtherVetting { get; set; }
        public string OtherVettingDescription { get; set; }
        public DateTime? OtherVettingDate { get; set; }
        public int? PaperworkStatusID { get; set; }
        public int? TravelDocumentStatusID { get; set; }
        public bool RemovedFromEvent { get; set; }
        public int? RemovalReasonID { get; set; }
        public int? RemovalCauseID { get; set; }
        public DateTime? DateCanceled { get; set; }
        public string Comments { get; set; }
        public int? ModifiedByAppUserID { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/training-event-student_item.ts


import { ITrainingEventStudent_Item } from './itraining-event-student_item';

export class TrainingEventStudent_Item implements ITrainingEventStudent_Item {
  
	public PersonID: number = 0;
	public TrainingEventID: number = 0;
	public IsVIP?: boolean;
	public IsParticipant?: boolean;
	public IsTraveling?: boolean;
	public DepartureCityID?: number;
	public DepartureDate?: Date;
	public ReturnDate?: Date;
	public VisaStatusID?: number;
	public HasLocalGovTrust?: boolean;
	public LocalGovTrustCertDate?: Date;
	public OtherVetting?: boolean;
	public PassedOtherVetting?: boolean;
	public OtherVettingDescription: string = "";
	public OtherVettingDate?: Date;
	public PaperworkStatusID?: number;
	public TravelDocumentStatusID?: number;
	public RemovedFromEvent: boolean = false;
	public RemovalReasonID?: number;
	public RemovalCauseID?: number;
	public DateCanceled?: Date;
	public Comments: string = "";
	public ModifiedByAppUserID?: number;
  
}






